Source/UI/SecurityCodeComponent/SecurityCodeComponent.swift

```diff
@@ -35,23 +45,25 @@ extension SecurityCodeComponent {
                         isSecurityCodeValid: @escaping (Bool) -> Void) {
     self.configuration = configuration
     self.isSecurityCodeValid = isSecurityCodeValid
-
-    self.cardValidator = CardValidator(environment: configuration.environment.checkoutEnvironment)
-
+    
+    cardValidator = CardValidator(environment: configuration.environment.checkoutEnvironment)
```

Comment:
I can see cardValidator has all the validators, for Security Component, can we inject only the `CVVValidator` only instead of cardValidator? In Android, we have independently added `CVVValidator` in this way we avoid unnecessary multiple validators for Securitycomponent's scope

Reply:
I'd seen your discussion under the Android repo and tried to do the same to our side. It meant refactoring for the `SecurityCodeViewModel` since it accepts `CardValidating` protocol conformant type at the moment. I didn't want to make that change because it'd introduce more moving pieces to this work and in the end the added value wouldn't worth that much of an update I decided. But thanks for this great shout! @chintan-soni-cko